staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -637,6 +637,9 @@ message WindowsContainerSecurityContext {
     // exist in the container image and be resolved there by the runtime;
     // otherwise, the runtime MUST return error.
     string run_as_username = 1;
+
+    // The contents of the GMSA credential spec to use to run this container.
+    string gmsa_credential_spec = 2;
```

Comment:
to align with the underlying runtime field this will get plumbed into, let's call this `credential_spec`

see https://github.com/opencontainers/runtime-spec/blob/5b71a03e2700cbb1091b78849b8748f5e1692650/schema/config-windows.json#L80

`pod.spec.securityContext.windowsOptions.gmsaCredentialSpec` is one possible source for this, but if we added more sources in the future, we wouldn't want to have to plumb all the sources through all the layers

Reply:
👍